companion/src/firmwares/customfunctiondata.cpp

```diff
@@ -199,18 +199,10 @@ QString CustomFunctionData::repeatToString(const int value)
 
 QString CustomFunctionData::enabledToString() const
 {
-  if ((func >= FuncOverrideCH1 && func <= FuncOverrideCHLast) ||
-      (func >= FuncAdjustGV1 && func <= FuncAdjustGVLast) ||
-      (func == FuncReset) ||
-      (func >= FuncSetTimer1 && func <= FuncSetTimerLast) ||
-      (func == FuncVolume) ||
-      (func == FuncBacklight) ||
-      (func <= FuncInstantTrim)) {
-    if (!enabled) {
-      return tr("DISABLED");
-    }
+  if (!enabled) {
+    return tr("DISABLED");
   }
-  return "";
+  return "ENABLED";
```

Comment:
This is only used by model print and is blank to save cluttering. This PR sets the default to enabled on all CFs so IMO keeping blank makes even more sense. IMO you really want the fact that a CF is disabled to be highlighted.

Reply:
I thought it was better to see status explicitly, but if you say so...

Reply:
Like most things visual it is often personal preference.
While testing found more unnecessary '0' values for functions which do not use the parameter. So I'll have to create a separate PR.

Reply:
If it were to stay it would need translation